log.js

```diff
@@ -13,7 +13,7 @@ function log(config){
   let server = new dwServer(host, username, password)
   
   let gotEntries = server.ls('../Logs');
-  let regex = new RegExp(`^${level}-blade`)
+  let regex = new RegExp(`^(custom)?${level}-blade`)
```

Comment:
I think the better place to change this is the default value for level above. If we change that to `(custom)?error` instead of just `error` then the tool still has sane defaults, but can be used to specifically log any given file if necessary.